detox/src/devices/common/drivers/android/exec/ADB.js

```diff
@@ -135,6 +132,19 @@ class ADB {
     await this.shell(deviceId, `am force-stop ${appId}`);
   }
 
+
+  async clearAppData(deviceId, packageId) {
+    return await this.shell(deviceId, `pm clear ${packageId}`, {verbosity: 'low', silent: true});
+  }
+
+  async createFileWithContent(deviceId, filepath, content) {
+    await this.shell(deviceId, `echo ${content} > ${filepath}`);
+  }
+
+  async readFile(deviceId, filepath) {
+    return await this.shell(deviceId, `cat ${filepath}`, {verbosity: 'low', silent: true}).catch(() => '');
```

Comment:
What is the reason to suppress errors here?

Reply:
This method is only used to read the hash file, and if the hash file doesn't exist I don't want that error to be shown to the user, since it just means that the apk needs to be reinstalled.